app/src/components/billing/BillingContentTabs/Invoices/Invoices.tsx

```diff
@@ -0,0 +1,58 @@
+import { Card, VStack, Text, Icon, Link as ChakraLink } from "@chakra-ui/react";
+import { useInvoices } from "~/utils/hooks";
+import InvoicesTable from "./InvoicesTable";
+import NewTestInvoiceButton from "./NewTestInvoiceButton";
+import { FaTable } from "react-icons/fa";
+
+const Invoices = () => {
+  const query = useInvoices(10000);
+
+  const pendingCount = query.data?.invoices.filter((inv) => inv.status === "PENDING").length;
+  const paidCount = query.data?.invoices.filter((inv) => inv.status !== "PENDING").length;
```

Comment:
I would rename this variable to something like "processedCount", since a REFUNDED invoice could count toward this total.

Reply:
I was thinking of the exact same name, but I thought it would be unclear.
Maybe "historyCount" will be even better.